src/fides/api/ops/models/privacy_request.py

```diff
@@ -812,6 +812,7 @@ class ProvidedIdentityType(EnumType):
     phone_number = "phone_number"
     ga_client_id = "ga_client_id"
     ljt_readerID = "ljt_readerID"
+    fides_user_device_id = "fides_user_device_id"
```

Comment:
Thanks very much for adding this @seanpreston, I'll start building off this work on my end. Something that could be added here too is adding this to the `Identity` enum

Reply:
Added here already https://github.com/ethyca/fides/pull/3132/files#diff-db878ff58b7521d21d47ca95444c66c319f0effe61468cc6e23976616655370eR29